src/views/pages/NewFeedMessage/Index.js

```diff
@@ -1,11 +1,17 @@
-import React from 'react';
+import React, {useState} from 'react';
 import PageContainer from '../../components/PageContainer';
 import PageTitle from '../../components/PageTitle';
-import Text from '../../components/Text';
 import Button from '../../components/Button';
 import FeedSubNav from '../../components/FeedSubNav';
+import Node from '../../../services/Node';
+import {useServiceContext} from '../../../contexts/ServiceContext';
 
 function NewFeedMessage() {
+  const {_, keymanager} = useServiceContext();
```

Comment:
I think `const {keymanager} = ..` will suffice without the `_`

Reply:
Won't that get back both the messageService _and_ the KeyManager?